.github/workflows/compare-chromium-versions.yml

```diff
@@ -0,0 +1,41 @@
+name: Compare Chromium versions
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  compare:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PR_NUMBER: ${{ github.event.number }}
+    steps:
+      - run: |
```

Comment:
I just want a normal description instead of this:

<img width="265" alt="image" src="https://github.com/user-attachments/assets/212b15a2-b388-4495-b0e6-b867e1d64dca">